cluster/images/etcd-empty-dir-cleanup/Dockerfile

```diff
@@ -12,12 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gliderlabs/alpine
+FROM busybox
```

Comment:
I'm not really sure what is best - we seem to have a mix of both, though unversioned `busybox` is generally more common.

I'm also generally less worried about CVEs affecting busybox, since historically there have been very few affecting it.